package/MDAnalysis/lib/mdamath.py

```diff
@@ -101,6 +101,7 @@ def angle(a, b):
     """
     x = np.dot(a, b) / (norm(a) * norm(b))
     # catch roundoffs that lead to nan otherwise
+    x = np.clip(x,-1.0,1.0)
```

Comment:
if we're clipping here then the if conditions below can never be triggered?

Reply:
Thanks @richardjgowers 
A silly mistake indeed! /